pkg/authorization/defaultrolebindings/project_policy.go

```diff
@@ -23,27 +23,59 @@ const (
 	DeployerServiceAccountName = "deployer"
 )
 
-func GetBootstrapServiceAccountProjectRoleBindings(namespace string) []rbacv1.RoleBinding {
+type projectRoleBindings func(namespace string) []rbacv1.RoleBinding
+type serviceAccountRoleBinding func(namespace string) rbacv1.RoleBinding
+
+func GetBootstrapServiceAccountProjectRoleBindings(namespace string) rbacv1.RoleBinding {
 	imagePuller := newOriginRoleBindingForClusterRoleWithGroup(ImagePullerRoleBindingName, ImagePullerRoleName, namespace, serviceaccount.MakeNamespaceGroupName(namespace))
 	imagePuller.Annotations[openShiftDescription] = "Allows all pods in this namespace to pull images from this namespace.  It is auto-managed by a controller; remove subjects to disable."
 
+	return imagePuller
+}
+
+func GetBuilderServiceAccountProjectRoleBindings(namespace string) rbacv1.RoleBinding {
 	imageBuilder := newOriginRoleBindingForClusterRoleWithSA(ImageBuilderRoleBindingName, ImageBuilderRoleName, namespace, BuilderServiceAccountName)
 	imageBuilder.Annotations[openShiftDescription] = "Allows builds in this namespace to push images to this namespace.  It is auto-managed by a controller; remove subjects to disable."
 
+	return imageBuilder
+}
+
+func GetDeployerServiceAccountProjectRoleBindings(namespace string) rbacv1.RoleBinding {
```

Comment:
(non-blocking): Add godoc

```suggestion
// GetDeployerServiceAccountProjectRoleBindings generates the role bindings that are specific to the "deployer" service account.
// These should only be created if the "DeploymentConfig" capability is enabled on the cluster.
func GetDeployerServiceAccountProjectRoleBindings(namespace string) rbacv1.RoleBinding {
```